### PR TITLE
Provide implicit method to convert Try to ErrorOr

### PR DESCRIPTION
**Pull request by Horneth**

### CODE REVIEW

File: src/main/scala/lenthall/validation/Validation.scala [hunk 1]
```diff
@@ -23,4 +24,10 @@ object Validation {
     case Success(result) => result.validNel
     case Failure(f) => f.getMessage.invalidNel
   }
+
+  implicit class TryValidation[A](val t: Try[A]) extends AnyVal {
+    def toErroOr: ErrorOr[A] = {
```
Comment (mcovarr) [inline src/main/scala/lenthall/validation/Validation.scala]: spelling fail

**Review by mcovarr — APPROVED** [review]
heck yeah, I'm actually going to use this in my WDL/WOM stuff :+1:

**Comment by mcovarr** [comment]
blargh :+1:

[Approved with PullApprove](https://pullapprove.com/broadinstitute/lenthall/pull-request/36/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/lenthall)

File: src/main/scala/lenthall/validation/Validation.scala (hunk 1 above)
Comment (Horneth) [reply src/main/scala/lenthall/validation/Validation.scala]: Jeez this PR has 1 method and I manage to fail spell it :(

**Comment by kshakir** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/lenthall/pull-request/36/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/lenthall)

**Pull request merged**